### ISSUE TITLE
Unchecking "No automatic rotation" is not working

### ISSUE THREAD

**Issue by maverick74** [zxing/zxing#1443]
Going into the App preferences and unchecking "No automatic rotation" is not working.
The app does not get into Portrait mode!

@srowen This works well in your archived version of BarcodeScanner+
can't that code me merged here?

**Comment by srowen** [comment]
That doesn't put it in portrait, just ignores rotation events. The main scan activity is landscape-only. I'm not even sure this preference really does anything on the latest android where you generally press the button to enable landscape switching.
This app is looong ago in maintenance mode and it's a lot of work to port the portrait support. I also can't update the app in Play anymore anyway.

**Comment by maverick74** [comment]
Oh...... OK!
I noticed that your version (Plus) is also archived hence i returned back here in hope this was still "alive".

I used to use initially this version and then yours (because of the rotation ability)... but since they're not in development anymore i'm going to have to search for an alternative (any OpenSource recommendation?)

It's sad. It was an amazing app that worked pretty well. 
**Thanks for all the time spent on it :)**
